macro/convertEvtCalc.py

```diff
@@ -21,6 +21,7 @@ def parse_file(infile):
     try:
         with open(infile, "r") as file:
             lines = file.readlines()
+            lines = lines[2:]
```

Comment:
Can this input file become larger than memory for large productions?

Reply:
The cases we have tested (large LLP mass, high multiplicity of final states~50) this does not happen. I do not see this changing in the future, given the limitation in the achievable LLP mass.